crates/sys/src/bindings.rs

```diff
@@ -2096,7 +2097,7 @@ extern "C" {
     pub fn furi_thread_get_current_id() -> FuriThreadId;
 }
 extern "C" {
-    #[doc = "Get FuriThread instance for current thread\n\nReturns:\n\n* FuriThread*\n\n"]
+    #[doc = "Get FuriThread instance for current thread\n\nReturns:\n\n* pointer to FuriThread or NULL if this thread doesn't belongs to Furi\n\n"]
```

Comment:
I believe this shouldn't affect our `Thread` wrapper type's usage of `ptr::NonNull`, as my analysis here should still hold: https://github.com/flipperzero-rs/flipperzero/pull/67#discussion_r1193164706

Reply:
While I can't think of any way to create a non-Furi thread, if the runtime doesn't perform a check, we should probably do so, if only to catch some future case where this call may return null for perhaps another reason.

Reply:
Switched from `NonNull::new_unchecked(...)` to `NonNull::new(...).expect(...)` for `furi_thread_get_current()`.